app/uploaders/splash_image_uploader.rb

```diff
@@ -0,0 +1,76 @@
+class SplashImageUploader < CarrierWave::Uploader::Base
```

Comment:
At a glance, this file seems like it might be a duplicate of `app/uploaders/footer_image_uploader.rb`.  If it isn't, is there low-hanging fruit for abstraction?

Reply:
Yes, they are in essence duplicates, but that's how Carrierwave wants you do it - one uploader per attribute.  Having said that, I'll see if it's possible to reuse the same one for all three images in `BrandingGroup`.

Reply:
Well whaddya know... it works!  Great suggestion.

Reply:
59fbd54